corehq/util/dates.py

```diff
@@ -81,3 +82,15 @@ def iso_string_to_datetime(iso_string):
     _assert(False, 'input not in expected format: {!r}'.format(iso_string))
     from dimagi.utils.parsing import string_to_utc_datetime
     return string_to_utc_datetime(iso_string)
+
+
+def iso_string_to_date(iso_string):
```

Comment:
I later found that a function with identical behavior exists within dimagi-utils' `loosechange`, but it didn't feel right to be just randomly pulling from those innards and/or refactor that code.